kortex_driver/src/hardware_interface.cpp

```diff
@@ -905,8 +918,19 @@ return_type KortexMultiInterfaceHardware::write(
       else
       {
         // Keep alive mode - no controller active
-        feedback_ = base_cyclic_.RefreshFeedback();
-        RCLCPP_DEBUG(LOGGER, "No controller active in LOW_LEVEL_SERVOING mode !");
+        try
+        {
+          feedback_ = base_cyclic_.RefreshFeedback();
+          RCLCPP_DEBUG(LOGGER, "No controller active in LOW_LEVEL_SERVOING mode !");
+        }
+        catch (k_api::KDetailedException & ex)
+        {
+          RCLCPP_ERROR_STREAM(LOGGER, "Kortex exception during Keep Alive Mode: " << ex.what());
+
+          RCLCPP_ERROR_STREAM(
+          LOGGER, "Error sub-code: " << k_api::SubErrorCodes_Name(
+                  k_api::SubErrorCodes((ex.getErrorInfo().getError().error_sub_code()))));
```

Comment:
and what happens after we caught that exception from the arm? can we keep operating? do we need to shut down? The original code burns everything down in this case.

Reply:
with this one I think its okay to keep operating since its waiting for the `joint_based_controller_running_` so we wont be sending any commands even if an exception occurs.

Reply:
normally we'd send the hardware component back into inactive state by returning an error from either read() or write() but maybe that level of detail would get in the way at the moment